File: space-game/Core.Tests/Data/MassTestsData.vb
Imports Core.Classes
Imports NUnit.Framework

Namespace Data
    Public Class MassTestsData
        Public Shared Function Equals_WhenComparing_ReturnsExpected_Data() As List(Of TestCaseData)
            Dim nullMass As Mass = Nothing
            Dim m = Mass.FromKilograms(12345)
            Dim copyOfM = m
            Dim notSame = Mass.FromKilograms(123456)

            Return New List(Of TestCaseData) From {
                New TestCaseData(m, nullMass, False),
                New TestCaseData(m, copyOfM, True),
                New TestCaseData(m, notSame, False)
            }
        End Function

    End Class
End Namespace

File: space-game/Core.Tests/Data/MassTestsData.cs
using Core.Classes;
using NUnit.Framework;
using System.Collections.Generic;

namespace Core.Tests.Data
{
    public class MassTestsData
    {
        public static List<TestCaseData> Equals_WhenComparing_ReturnsExpected_Data()
        {
            Mass nullMass = null;
            Mass m = Mass.FromKilograms(12345);
            Mass copyOfM = m;
            Mass notSame = Mass.FromKilograms(123456);

            return new List<TestCaseData> {
                new TestCaseData(m, nullMass, false),
                new TestCaseData(m, copyOfM, true),
                new TestCaseData(m, notSame, false)
            };
        }
    }
}
